pkg/image/imageversion.go

```diff
@@ -36,8 +39,12 @@ var (
 const (
 	// ConformanceImageVersionAuto represents detecting the server's kubernetes version.
 	ConformanceImageVersionAuto = "auto"
-	// ConformanceImageVersionLatest represents always using the server's latest version.
+	// ConformanceImageVersionLatest represents always using the server's latest dev version.
 	ConformanceImageVersionLatest = "latest"
+
+	// DevVersionURL is the URL which should respond with a simple text of the latest version for devs.
+	DevVersionURL      = "https://storage.googleapis.com/k8s-release-dev/ci/latest.txt"
```

Comment:
@johnSchnake what happens if those urls change or go away? Can user overwrite with a flag (i know we already got so much flags).

Reply:
You can always specify an image directly with an existing flag. I dont think we need to provide a flag to separately change this URL. If it all changes then we patch sonobuoy is all, which is the same thing we'd have to do even if we had this set-but-customizable.